financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheet.kt

```diff
@@ -72,6 +76,24 @@ class FinancialConnectionsSheet internal constructor(
             )
         }
 
+        /**
+         * Create a [FinancialConnectionsSheet] used for Jetpack Compose.
+         *
+         * This API uses Compose specific API [rememberLauncherForActivityResult] to register a
+         * [androidx.activity.result.ActivityResultLauncher] into the current activity,
+         * so it should be called as part of Compose initialization path.
+         */
+        @Composable
+        fun createComposable(
+            callback: (FinancialConnectionsSheetResult) -> Unit
+        ): FinancialConnectionsSheet = FinancialConnectionsSheet(
+            FinancialConnectionsSheetForDataLauncher(
+                rememberLauncherForActivityResult(
+                    FinancialConnectionsSheetForDataContract()
+                ) { callback(it) }
+            )
+        )
```

Comment:
Alternatively we could use the same naming we do in the [Payments SDK](https://github.com/stripe/stripe-android/blob/994b3f1b/payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt#L80). 

However, this is not the Compose naming convention. 

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/99293320/171966154-c3169d51-eb4c-40e6-8341-4a49c1c80922.png">

https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#naming-composable-functions-that-remember--the-objects-they-return